build.sbt

```diff
@@ -92,6 +108,7 @@ lazy val journal = (project in file("journal")
     `cassandra-driver`,
     scassandra,
     scache,
+    `cassandra-sync`,
```

Comment:
is it used?

Reply:
No, it is not used. I was the one removing it, initially, in previous commits.

I have added it back, because lack of it breaks the backwards compatibility.

I.e. the library depending on `kafka-journal` will not get it anymore.

MiMa detects it and complains.